src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/StaticWebAssetsBaselines/Publish_DoesNotGenerateManifestJson_IncludesJSModulesOnBlazorBootJsonManifest.Publish.files.json

```diff
@@ -128,9 +128,6 @@
   "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\System.Private.CoreLib.dll",
   "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\System.Private.CoreLib.dll.br",
   "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\System.Private.CoreLib.dll.gz",
-  "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\System.Runtime.InteropServices.JavaScript.dll",
-  "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\System.Runtime.InteropServices.JavaScript.dll.br",
-  "${ProjectPath}\\bin\\Debug\\${Tfm}\\publish\\wwwroot\\_framework\\System.Runtime.InteropServices.JavaScript.dll.gz",
```

Comment:
Why has this changed? The 6.0 output shouldn't be different now should it?

Reply:
Oh never mind, I see it's just the ordering that's changed. I'm sure that's not breaking.

Reply:
Yeah, this is an artifact of the way we order things in the baseline